apps/condo/domains/billing/access/BillingRecipient.js

```diff
@@ -0,0 +1,53 @@
+// @ts-nocheck
+
+/**
+ * Generated by `createschema billing.BillingRecipient 'organization:Relationship:Organization:CASCADE; tin:Text; iec:Text; bic:Text; context?:Relationship:BillingIntegrationOrganizationContext:SET_NULL; bankAccount:Text; name?:Text; approved:Checkbox; meta?:Json;'`
+ */
+
+const { get } = require('lodash')
+
+const { getById } = require('@core/keystone/schema')
+
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+
+const { canReadBillingEntity, checkBillingIntegrationAccessRight } = require('@condo/domains/billing/utils/accessSchema')
+
+
+async function canReadBillingRecipients ({ authentication }) {
+    return await canReadBillingEntity(authentication)
+}
+
+async function canManageBillingRecipients ({ authentication, operation, originalInput, itemId, listKey, context }) {
+    const { item: user } = authentication
```

Comment:
May be should move it to function arguments decompostion? Since arguments are partly decomposed here and there...Which is not clean I guess